Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
```

Comment:
interesting. so rust uses a toml file to keep track of dependencies and stuff?

Reply:
Yes, it's used as the manifest file. I'm not fully familiar with them, but that is their general purpose